base_layer/core/tests/chain_storage_tests/chain_storage.rs

```diff
@@ -1289,7 +1301,10 @@ fn invalid_block() {
 
     // Cleanup test data - in Windows the LMBD `set_mapsize` sets file size equals to map size; Linux use sparse files
     if std::path::Path::new(&temp_path).exists() {
-        std::fs::remove_dir_all(&temp_path).unwrap();
+        match std::fs::remove_dir_all(&temp_path) {
```

Comment:
see review comment

Reply:
The tests failed in Windows under certain system curcimstances, so this had to be done to make the unit tests resilient. For some reason the test folders could not be deleted, so the code panicked at the `.unwrap()`, so this change was necessary.